packages/design-system/src/components/Form/Primitives/Primitives.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import Input from './Input/Input';
+import Textarea from './Textarea/Textarea';
+import Select from './Select/Select';
+import { action } from '@storybook/addon-actions';
+import Dropdown from '../../Dropdown';
+import { AffixButton } from '../FieldGroup/Affix';
+import { StackVertical } from '../../Stack';
+
+export default {
+	component: Textarea,
+};
+
+export const TextareaPrimitive = () => (
+	<StackVertical gap="XS" padding="XS" align="stretch" justify="start">
+		<Textarea placeholder="Placeholder" />
+		<Textarea defaultValue="With default value" />
+		<Textarea defaultValue="Test read-only" readOnly />
+		<Textarea defaultValue="Test disabled" disabled />
+		<Textarea placeholder="Test disabled empty" disabled />
+	</StackVertical>
+);
+
+export const InputPrimitive = () => (
+	<StackVertical gap="XS" padding="XS" align="stretch" justify="start">
+		<Input type="text" placeholder="Placeholder" />
+		<Input type="text" defaultValue="With default value" />
+		<Input type="text" readOnly disabled defaultValue="Test read-only" />
+		<Input type="text" disabled defaultValue="Test disabled" />
+		<Input type="text" disabled placeholder="Test disabled empty" />
+		<Input
+			placeholder="Test with afixes"
+			type="text"
+			prefix={{ type: 'text', children: 'Affix', hideText: false, icon: 'talend-file-o' }}
+			suffix={{
+				type: 'button',
+				onClick: () => action('clicked'),
+				children: 'action',
+				icon: 'talend-file-o',
+			}}
+		/>
+	</StackVertical>
+);
+
+export const InputPrimitivePassword = () => (
+	<StackVertical gap="XS" padding="XS" align="stretch" justify="start">
+		<Input type="password" placeholder="Placeholder" />
+		<Input type="password" defaultValue="With default value" />
+		<Input type="password" readOnly defaultValue="Test read-only" />
+		<Input type="password" disabled defaultValue="Test disabled" />
+		<Input type="password" disabled placeholder="Test disabled empty" />
+		<Input
+			placeholder="Test with afixes"
+			type="password"
+			prefix={{ type: 'text', children: 'Affix', hideText: false, icon: 'talend-file-o' }}
+			suffix={{
+				type: 'button',
+				onClick: () => action('clicked'),
+				children: 'action',
+				icon: 'talend-file-o',
+			}}
+		/>
+	</StackVertical>
+);
+
+export const SelectPrimitive = () => {
```

Comment:
Caret looks really close to the border 

<img width="938" alt="Screenshot 2022-05-16 at 11 49 15" src="https://user-images.githubusercontent.com/58977230/168566504-5977f1c6-2f99-433f-894d-13ca07128c08.png">

Reply:
Right. I haven't styled the selects yet. I should, thanks!

Reply:
Updated

![Screenshot 2022-05-16 at 17 10 53](https://user-images.githubusercontent.com/1319432/168625175-638d7286-2c75-4e55-af0f-a0182044c2cd.png)